bundles/org.openhab.transform.map/src/test/java/org/openhab/transform/map/internal/MapTransformationServiceTest.java

```diff
@@ -125,7 +122,10 @@ public Void call() throws Exception {
                 }
             }, 10000, 100);
         } catch (IOException e1) {
-            e1.printStackTrace(System.err);
+            PrintStream err = System.err;
+            if (err != null) {
+                e1.printStackTrace(err);
+            }
```

Comment:
System.err can be null!?

Reply:
https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/19fb8f93c59dfd791f62d41f332db9e306bc1422/src/java.base/share/classes/java/lang/System.java#L155